lib/spack/spack/util/editor.py

```diff
@@ -39,8 +40,13 @@ def _find_exe_from_env_var(var):
     if not exe:
         return None, []
 
+    if sys.platform == "win32":
+        # Fix separators
+        exe = exe.replace('\\', '/')
```

Comment:
Is this still needed now that https://github.com/spack/spack/pull/28402 is merged?

Reply:
Nope! Refactored.